release.toml

```diff
@@ -1,7 +1,7 @@
-pre-release-commit-message = "Release: {{version}}"
 post-release-commit-message = "Start next development iteration: {{next_version}}"
-tag-message = "{{tag_name}}"
+pre-release-commit-message = "release: {{crate_name}}: version {{version}}"
```

Comment:
This is mostly stealing the config from Swift CLI which has been working well, this config has had a few issues in the past

Reply:
It will create a commit for each crate and then publish them in the correct order according to who's dependent on whom.